stackdriver-core/lib/stackdriver/core/async_actor.rb

```diff
@@ -199,6 +199,30 @@ def async_stop!
       end
 
       ##
+      # Block current thread until the async job is fully stopped.
```

Comment:
And I think the method is useful to clients and should be part of the interface.